match.py

```diff
@@ -0,0 +1,177 @@
+"""Switch match."""
+
+import ipaddress
+
+from pyof.v0x01.common.flow_match import FlowWildCards
+
+from kytos.core import log
+
+from .settings import IPV4
+
+
+def format_request(request):
+    """Format user request to match function format."""
+    args = {}
+    for key, value in request.items():
+        args[key] = value
+    return args
+
+
+def do_match(flow_to_install, version, stored_flow_dict):
+    """Match a packet against this flow."""
+    if version == 0x01:
+        return match10(stored_flow_dict, flow_to_install)
+    elif version == 0x04:
+        return match13_no_strict(flow_to_install, stored_flow_dict)
+    return None
+
+
+def _get_match_fields(flow_dict):
+    """Generate match fields."""
+    match_fields = {}
+    if 'match' in flow_dict:
+        for key, value in flow_dict['match'].items():
+            match_fields[key] = value
+    return match_fields
+
+
+# pylint: disable=too-many-return-statements, too-many-statements, R0912
+def match10(flow_dict, args):
+    """Match a packet against this flow (OF1.0)."""
+    log.debug('Matching packet')
+    match_fields = _get_match_fields(flow_dict)
+    if not match_fields.get('wildcards') & FlowWildCards.OFPFW_IN_PORT:
```

Comment:
```suggestion
    wildcards = match_fields.get('wildcards')
    if not wildcards & FlowWildCards.OFPFW_IN_PORT:
```

Replace all the `match_fields.get('wildcards')` below.